airbyte-integrations/connectors/source-mixpanel/source_mixpanel/components.py

```diff
@@ -0,0 +1,330 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
```

Comment:
Could you please add tests for each custom component?

Reply:
components.y if covered by 90%